.github/workflows/test-convene-web.yml

```diff
@@ -53,7 +53,7 @@ jobs:
         run: yarn install
 
   test-rspec:
-    name: Run tests
+    name: Run Rspec tests
```

Comment:
Does this pipeline rename need to be added in another place?

![oops](https://github.com/zinc-collective/convene/assets/16140873/4a6390f7-a3be-4f55-9a70-1f5a698b6c32)